src/backend/distributed/commands/extension.c

```diff
@@ -510,25 +510,98 @@ MarkExistingObjectDependenciesDistributedIfSupported()
 	Oid citusTableId = InvalidOid;
 	foreach_oid(citusTableId, citusTableIdList)
 	{
+		if (IsTableOwnedByExtension(citusTableId))
```

Comment:
I think we should have done that even before view, but we don't need to backport it to the 11 Beta as we'll provide complete support with 11, right?

Reply:
Hmm, actually backporting this pieces could make sense, though not urgent.

So, I can create a separate commit and cherry-pick to 10.0+.

Does that sound good?